test_requests.py

```diff
@@ -1062,6 +1062,13 @@ def test_json_param_post_content_type_works(self):
         assert 'application/json' in r.request.headers['Content-Type']
         assert {'life': 42} == r.json()['json']
 
+    def test_json_param_post_should_not_override_data_param(self):
+        r = requests.Request(method='POST', url='http://httpbin.org/post',
+                             data={'stuff'.encode('utf-8'): 'elixr'},
```

Comment:
Copy-paste from the other testcase! :) Will remove.

On Wednesday 09 September 2015 04:58 AM, Ian Cordasco wrote:
>
> In test_requests.py 
> <https://github.com/kennethreitz/requests/pull/2763#discussion_r38992569>:
>
> > @@ -1062,6 +1062,13 @@ def test_json_param_post_content_type_works(self):
> >          assert 'application/json' in r.request.headers['Content-Type']
> >          assert {'life': 42} == r.json()['json']
> >
> > +    def test_json_param_post_should_not_override_data_param(self):
> > +        r = requests.Request(method='POST', url='http://httpbin.org/post',
> > +                             data={'stuff'.encode('utf-8'): 'elixr'},
>
> Why are we calling |str.encode|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/kennethreitz/requests/pull/2763/files#r38992569>.
>